client/js/verification-portal/components/get-resource-to-verify-button.js

```diff
@@ -103,62 +103,16 @@ const populateForm = ({ changeFieldValue, resource }) => {
 		'verifiedData.services.verified',
 		_.get(verifiedData, 'services.verified'),
 	)
-	changeFieldValue(
-		'hours[0].open',
-		convertNumberToTimeFormat(_.get(hours, '[0].open')),
-	)
-	changeFieldValue(
-		'hours[0].close',
-		convertNumberToTimeFormat(_.get(hours, '[0].close')),
-	)
-	changeFieldValue(
-		'hours[1].open',
-		convertNumberToTimeFormat(_.get(hours, '[1].open')),
-	)
-	changeFieldValue(
-		'hours[1].close',
-		convertNumberToTimeFormat(_.get(hours, '[1].close')),
-	)
-	changeFieldValue(
-		'hours[2].open',
-		convertNumberToTimeFormat(_.get(hours, '[2].open')),
-	)
-	changeFieldValue(
-		'hours[2].close',
-		convertNumberToTimeFormat(_.get(hours, '[2].close')),
-	)
-	changeFieldValue(
-		'hours[3].open',
-		convertNumberToTimeFormat(_.get(hours, '[3].open')),
-	)
-	changeFieldValue(
-		'hours[3].close',
-		convertNumberToTimeFormat(_.get(hours, '[3].close')),
-	)
-	changeFieldValue(
-		'hours[4].open',
-		convertNumberToTimeFormat(_.get(hours, '[4].open')),
-	)
-	changeFieldValue(
-		'hours[4].close',
-		convertNumberToTimeFormat(_.get(hours, '[4].close')),
-	)
-	changeFieldValue(
-		'hours[5].open',
-		convertNumberToTimeFormat(_.get(hours, '[5].open')),
-	)
-	changeFieldValue(
-		'hours[5].close',
-		convertNumberToTimeFormat(_.get(hours, '[5].close')),
-	)
-	changeFieldValue(
-		'hours[6].open',
-		convertNumberToTimeFormat(_.get(hours, '[6].open')),
-	)
-	changeFieldValue(
-		'hours[6].close',
-		convertNumberToTimeFormat(_.get(hours, '[6].close')),
-	)
+	_.forEach([0, 1, 2, 3, 4, 5, 6], (hour, i) => {
```

Comment:
@bhavyaab Would you mind adding a comment here to make it a little more clear? Something like "Populate the hours for all days of the week".

Reply:
will add that